src/utils/constants.js

```diff
@@ -1,7 +1,14 @@
+import moment from 'moment';
```

Comment:
Currently `moment` is a legacy project in maintenance mode. Dev community proposes to use alternative libs such as `date-fns`, `day.js`, etc.

Reply:
I will add a task for it, thanks for informing